TKGame/TKGame.cs

```diff
@@ -200,7 +200,12 @@ protected override void Draw(GameTime gameTime)
             spriteBatch.Begin(SpriteSortMode.Deferred, BlendState.NonPremultiplied);
 
             //Draws the image into the Background
-            spriteBatch.Draw(Art.BackgroundTexture, levelComponent.GetCurrentStage().Background.BackgroundRect, Color.White);
+            if(levelComponent.GetCurrentStage().stageName == "room0.json")
+                spriteBatch.Draw(Art.BackgroundTexture1, levelComponent.GetCurrentStage().Background.BackgroundRect, Color.White);
+            else if (levelComponent.GetCurrentStage().stageName == "room1.json")
+                spriteBatch.Draw(Art.BackgroundTexture2, levelComponent.GetCurrentStage().Background.BackgroundRect, Color.White);
+            else
+                spriteBatch.Draw(Art.BackgroundTexture3, levelComponent.GetCurrentStage().Background.BackgroundRect, Color.White);
```

Comment:
The background shouldn't be handled by the world class, stages should handle setting their own background (probably best to add it to the stage json information).
You'll have to update the Load and Save JSON function and make another class for the background data (like what I did for the Triggers/Entities) so that when a stage is loaded it sets the background to w/e the JSON had.
This way it will automatically be drawn when the stage is drawn, not by adding calls in the TKGame.Draw() function.